### DISCUSSION TITLE
collection/linkedlist

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#93] [Announcements]
# collection/linkedlist

Java程序员进阶之路，小白的零基础Java教程，Java LinkedList详解

https://tobebetterjavaer.com/collection/linkedlist.html

**Comment by gt-liu** [comment]
怎么没有时间复杂度的说法了呢

**Reply by itwanger** [reply]
新的版本会加上，这周末会更新一版

**Comment by icc0216** [comment]
添加第二个元素的图没看懂，first结点怎么同时指向王二和王三

**Comment by ayu255** [comment]
这个图是不是有点混淆了，first和last不是node的引用吗，怎么同时指向王二王三。第一张图里怎么把first和last当初node结点了。

**Comment by CharlesShan-hub** [comment]
"Josh Bloch是 Java 集合框架的作者"这个图片挂了
